src/Mono.Android/Android.Runtime/AndroidRuntime.cs

```diff
@@ -246,7 +308,7 @@ public override void RemovePeer (IJavaPeerable value)
 
 		public override IJavaPeerable PeekPeer (JniObjectReference reference)
```

Comment:
Is `PeekPeer()` actually being used by something, which is why it needs to be implemented?

Reply:
It is used from the marshal methods like this:
```
android.runtime.JavaProxyThrowable: System.NotImplementedException: The method or operation is not implemented.
  at Android.Runtime.AndroidValueManager.PeekPeer (Java.Interop.JniObjectReference reference) [0x00001]
  at Java.Interop.JniRuntime+JniValueManager.PeekValue (Java.Interop.JniObjectReference reference) [0x0002f]
  at Java.Interop.JniRuntime+JniValueManager.GetValue[T] (Java.Interop.JniObjectReference& reference, Java.Interop.JniObjectReferenceOptions options, System.Type targetType)
  at Java.Interop.JniRuntime+JniValueManager.GetValue[T] (System.IntPtr handle) [0x0000a]
  at xatemplateaot.MainActivity+__<$>_jni_marshal_methods.n_onCreate_Landroid_os_Bundle_ (System.IntPtr __jnienv, System.IntPtr __this, System.IntPtr savedInstanceState) [0x0001e]
       at md58018c7d08c228c8e2a3e03723c59ca27.MainActivity.n_onCreate(Native Method)
       at md58018c7d08c228c8e2a3e03723c59ca27.MainActivity.onCreate(MainActivity.java:29)
```